docs/marketplace/add_to_marketplace/manage_marketplace_items.md

```diff
@@ -0,0 +1,117 @@
+---
+id: manage_marketplace_items
+title:  Manage Marketplace Items
+sidebar_label:  Manage Marketplace Items
+---
+
+There are three main methods to create, modify and delete Marketplace items:
+
+* (recommended) Use [`miactl`](/cli/miactl/10_overview.md), the Mia-Platform command line interface tool.
+* Open an issue on Mia-Platform [Github community page](https://github.com/mia-platform/community).
+* (deprecated) Use the [CMS](/business_suite/guide_cms.md).
+
+## Use `miactl`
+
+
+First of all, you need to setup `miactl`, as explained in the [dedicated doc](/cli/miactl/20_setup.md).
+
+With the `miactl marketplace` subcommands, you can perform several actions, described here below.
+
+### Create an item 
```

Comment:
I would put here below a brief introduction with a business use case (e.g. ACME Company decides to create an item on the Marketplace: why? --> link to another doc page. how? --> provide here below a practical example with Miactl)